src/App.tsx

```diff
@@ -1,12 +1,46 @@
 import React from 'react';
 import { BrowserRouter, Link, Route, Routes } from 'react-router-dom';
 import './App.css';
+import Button from './components/Button';
+import Dropdown from './components/Dropdown';
 import Station from './components/Station';
 import Stations from './components/Stations';
 
-const App = () => (
-    <BrowserRouter>
+const stationMap = new Map<string, string>([
+    ['Kings Cross', 'KGX'],
+    ['Edinburgh', 'EDB'],
+    ['Victoria', 'VIC'],
+    ['Birmingham', 'BHI'],
+    ['Glasgow', 'GLC'],
+]);
+
+const stationNames =  Array.from(stationMap.keys());
+
+const urlMaker = (stationOne: string, stationTwo: string) => {
+    return `https://www.lner.co.uk/travel-information/travelling-now/live-train-times/depart/${stationMap.get(stationOne)}/${stationMap.get(stationTwo)}/`;
+};
+
+const App = () => {
+
+    const [departureStation, setDepartureStation] = React.useState('');
+    const [arrivalStation, setArrivalStation] = React.useState('');
+
+    const onSubmit = () => {
+        departureStation && arrivalStation && window.open(urlMaker(departureStation, arrivalStation));
+    };
+
+    return <BrowserRouter>
         <div className = "App">
+            <div className = "dropdown-menus-container">
+                <div className = "dropdown">
+                    <Dropdown valueUpdateFunction = { setDepartureStation } placeHolder = "Select Station" label = 'Departure:' selectableStations = { stationNames } id = 'departure-station-selection'/>
+                </div>
+                <div className = "dropdown">
+                    <Dropdown valueUpdateFunction = { setArrivalStation } placeHolder = "Select Station" label = 'Arrival:' selectableStations = { stationNames.filter((s) => s!==departureStation) } id = 'arrival-station-selection'/>
+                </div>
+            </div>
+            
+            <Button text = 'Select Station' onClick = { onSubmit } classes = 'is-danger' disabled = { !departureStation || !arrivalStation }/>
```

Comment:
What is is-danger being used for?

Reply:
'is-danger' is the class name for the bulma CSS library:
https://bulma.io/documentation/elements/button/#colors